Add `Skin of the Year 2024: Voting Open` news post (#12797)
ppy/osu-wiki: news/2025/2025-02-11-skin-of-the-year-2024.md

- In September, star fountains and star bursts were added to osu!(lazer) with full skinning support.
- In October, some elements received extra customisation, allowing the aspect ratio to be customised for the avatar skin elements, and other skin elements got the option of changing their colour in-game.
- In November, the "relative size" option was added for the song progress bar skin element, which means the size will automatically adjust with the game's resolution.

Now that we've gone through everything skinning related that happened in 2024, let's go into the details of the contest itself.

## Voting details

All skins released in 2024 and added to the skinship compendium are eligible, **including contest entries (regardless of their final placement).** This year you'll have **12** votes to cast, so be sure to use them wisely!

The top 3 entries will all receive a unique profile badge based on their entry along with 4, 3 and 2 months of osu!supporter respectively. Entries #4–10 will all receive a month of osu!supporter each.

> In total there have been **233** skins submitted this year across all forum categories ([work in progress](https://osu.ppy.sh/community/forums/119), [remixed](https://osu.ppy.sh/community/forums/124), [completed](https://osu.ppy.sh/community/forums/109)), with **105** of them getting added to the [skinship compendium](https://compendium.skinship.xyz/). As always, it's highly recommended that you peruse through the compendium, and pick out your 2024 favourites!

Reviewer: ```suggestion
In total, there have been **233** skins submitted this year across all forum categories ([work in progress](https://osu.ppy.sh/community/forums/119), [remixed](https://osu.ppy.sh/community/forums/124) and [completed](https://osu.ppy.sh/community/forums/109)), with **105** of them in the [skinship compendium](https://compendium.skinship.xyz/). As always, we highly recommended that you peruse through the compendium or view the skins in-game before picking out your 2024 favourites!
```

[**Click here to get involved and vote for 2024's Skin of the Year!**](https://osu.ppy.sh/community/contests/239)

> The voting will be up for 14 days after this post goes up, and will close on the 25th at 23:59 UTC, so don't hesitate too long and get your votes in on time!

Reviewer: ```suggestion
The voting will be up for **14 days** after this post goes up, and will close on the 25th at 23:59 UTC, so don't hesitate too long and get your votes in on time!
```
